webmails/roundcube/config.inc.php

```diff
@@ -34,6 +34,10 @@
 $config['managesieve_host'] = $imap;
 $config['managesieve_usetls'] = false;
 
+// Customization settings
+$config['support_url'] = getenv('WEB_ADMIN') ? '../..' . getenv('WEB_ADMIN') : '';
```

Comment:
Why do you set the URL where the webmail user is supposed to find support for their webmail experience to the admin interface? I'm not sure this would help anyone :/

Reply:
It's not the support for the "webmail experience", it's more that you find out about Mailu Admin Page (settings, information). And there is also a another link that redirects to the Mailu documentation for help. I found this very use- and helpful.

Reply:
Oh… sorry, then I missunderstood the comment in roundcubes default config.